example_configs/dex_config.yml

```diff
@@ -0,0 +1,25 @@
+# lldap configuration:
+# LLDAP_LDAP_BASE_DN:    dc=example,dc=com
+
+# ##############################
+# rest of the Dex options
+# ##############################
+
+connectors:
+  - type: ldap
+    id: ldap
+    name: LDAP
+    config:
+      host: lldap-host # make sure it does not start with `ldap://`
+      port: 3890
+      insecureNoSSL: true
+      insecureSkipVerify: true
+      bindDN: uid=admin,ou=people,dc=example,dc=com # replace admin with your admin user
+      bindPW: very-secure-password # replace with your admin password
+      userSearch:
+        baseDN: ou=people,dc=example,dc=com
+        username: uid
+        idAttr: uid
+        emailAttr: mail
+        nameAttr: displayName
+        preferredUsernameAttr: uid
```

Comment:
You don't want to add the group config? It should probably look like that:

```yaml
    groupSearch:
      baseDN: ou=groups,dc=example,dc=com
      filter: "(objectClass=groupOfUniqueNames)"
      userMatchers:
      - userAttr: uid
        groupAttr: member
      nameAttr: displayName
```

Reply:
I did not have to add this to my config for dex to function properly. Is this required?

Reply:
It's only required if you want to expose the groups to other apps, but it would be useful (e.g. only allow access to service X if the user is in group `x_users`).
I think it would be valuable to add to the example config, if you can confirm that it works.

I don't know much about Dex, maybe they have a way in their UI to see the user's groups that you could use to confirm that it works?

Reply:
Yes it works will update.